### DISCUSSION TITLE
Search Database in all Connections

### DISCUSSION THREAD

**Discussion by marvin-me** [dbeaver/dbeaver#21243] [Q&A]
I'm using DBeaver to connect to multible MSSQL Server. Each Connection contains multible Databases. I can't seem to find a function to search for a Database in all open Connections. I can use "expand all" in the databasenavigator and filter for databases but DBeaver will expand all databases and tables. With many databases this freezes the Programm.
Is there a function to search for a specific database?
Thank you for your help.

**Comment by E1izabeth** [comment]
Have you tried using this dialog with `Ctrl+Shift+D`?
![image](https://github.com/dbeaver/dbeaver/assets/28875055/f6735f15-bc60-486c-a66b-585569e731ac)

Or do you need to find only database node in navigator?

**Reply by marvin-me** [reply]
Unfortunately this dialog is only able to find tables in a single connection. I don't need the node in the navigator specifically, but a function to quickly jump to the database im looking for. Maybe something like `STRG+F` in the Databasenavigator.

**Reply by E1izabeth** [reply]
I see. Probably it's a good idea. Could you create a new issue about it?
